packages/angular/projects/tx-native-angular-sdk/tests/language-picker.component.spec.ts

```diff
@@ -63,7 +53,7 @@ describe('LanguagePickerComponent', () => {
     spyOn(service, 'getLanguages').and.resolveTo(languages);
 
     // act
-    await component.ngOnInit();
+    await component.getLanguages();
```

Comment:
Here I would like to test the ngOnInit event, in order to test if the languages are loaded correctly at that stage. Is there any reason for replacing it?